docs/toc.json

```diff
@@ -25,7 +25,8 @@
 			{ "docs/designers-developers/developers/block-api/block-transforms.md": [] },
 			{ "docs/designers-developers/developers/block-api/block-templates.md": [] },
 			{ "docs/designers-developers/developers/block-api/block-patterns.md": [] },
-			{ "docs/designers-developers/developers/block-api/block-annotations.md": [] }
+			{ "docs/designers-developers/developers/block-api/block-annotations.md": [] },
+			{ "docs/designers-developers/developers/block-api/versions.md": [] }
```

Comment:
Shouldn't it be earlier in the menu, maybe after `block-attributes.md`?

Reply:
The idea for me was that the other docs should reflect the last API version and this document is just some kind of "reference" to all the previous versions. (so less important)

Reply:
Make sense, I didn't think of it this way 👍🏻